### ISSUE TITLE
java.lang.UnsupportedOperationException when playing age restriced videos

### ISSUE THREAD

**Issue by l0c4lh057** [sedmelluq/lavaplayer#163]
Whenever I try to play an age restricted youtube video I get an exception.
I am using Discord4J v2.10.1 and Lavaplayer v1.3.11. First I used Lavaplayer v1.3.10 and it didn't work either but I also got no error.

When trying to play an age restricted video the bot says it's playing the song and then instantly skips it.

I'm not sure if this is a D4J problem or a Lavaplayer problem.
And if it is a D4J problem: I know that there is D4J v3, but I need some time until I will have remade my bot in v3.

Full stacktrace:
```
22:43:57.522: [ERROR][D4J AudioThread][sx.blah.discord.Discord4J] - Discord4J Internal Exception
java.lang.UnsupportedOperationException
	at com.sedmelluq.discord.lavaplayer.track.playback.TerminatorAudioFrame.getVolume(TerminatorAudioFrame.java:18)
	at com.sedmelluq.discord.lavaplayer.track.playback.AllocatingAudioFrameBuffer.filterFrame(AllocatingAudioFrameBuffer.java:194)
	at com.sedmelluq.discord.lavaplayer.track.playback.AllocatingAudioFrameBuffer.provide(AllocatingAudioFrameBuffer.java:61)
	at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.provide(LocalAudioTrackExecutor.java:398)
	at com.sedmelluq.discord.lavaplayer.track.BaseAudioTrack.provide(BaseAudioTrack.java:92)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayer.provide(DefaultAudioPlayer.java:189)
	at com.sedmelluq.discord.lavaplayer.track.playback.AudioFrameProviderTools.delegateToTimedProvide(AudioFrameProviderTools.java:18)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayer.provide(DefaultAudioPlayer.java:175)
	at Lavaplayer.AudioProvider.isReady(AudioProvider.java:27)
	at sx.blah.discord.handle.audio.impl.DefaultProcessor.isReady(DefaultProcessor.java:42)
	at sx.blah.discord.handle.audio.impl.AudioManager.getAudioDataForProvider(AudioManager.java:217)
	at sx.blah.discord.handle.audio.impl.AudioManager.sendAudio(AudioManager.java:152)
	at sx.blah.discord.api.internal.UDPVoiceSocket.lambda$new$0(UDPVoiceSocket.java:116)
	at sx.blah.discord.api.internal.UDPVoiceSocket.lambda$begin$2(UDPVoiceSocket.java:233)
	at sx.blah.discord.util.HighPrecisionRecurrentTask.run(HighPrecisionRecurrentTask.java:60)
```

**Comment by KyuBlade** [comment]
I have the same issue out of D4j and it's not a restricted video.

`Exception in thread "Playback thread" java.lang.UnsupportedOperationException
	at com.sedmelluq.discord.lavaplayer.track.playback.TerminatorAudioFrame.getVolume(TerminatorAudioFrame.java:18)
	at com.sedmelluq.discord.lavaplayer.track.playback.AllocatingAudioFrameBuffer.filterFrame(AllocatingAudioFrameBuffer.java:194)
	at com.sedmelluq.discord.lavaplayer.track.playback.AllocatingAudioFrameBuffer.provide(AllocatingAudioFrameBuffer.java:61)
	at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.provide(LocalAudioTrackExecutor.java:398)
	at com.sedmelluq.discord.lavaplayer.track.BaseAudioTrack.provide(BaseAudioTrack.java:92)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayer.provide(DefaultAudioPlayer.java:189)
	at com.sedmelluq.discord.lavaplayer.format.AudioPlayerInputStream.attemptRetrieveFrame(AudioPlayerInputStream.java:110)
	at com.sedmelluq.discord.lavaplayer.format.AudioPlayerInputStream.ensureAvailable(AudioPlayerInputStream.java:94)
	at com.sedmelluq.discord.lavaplayer.format.AudioPlayerInputStream.read(AudioPlayerInputStream.java:56)
	at javax.sound.sampled.AudioInputStream.read(AudioInputStream.java:292)
	at javax.sound.sampled.AudioInputStream.read(AudioInputStream.java:232)
	at com.arthium.musica.audio.DesktopAudioPlayer.runPlayback(DesktopAudioPlayer.kt:86)
	at com.arthium.musica.audio.DesktopAudioPlayer.access$runPlayback(DesktopAudioPlayer.kt:14)
	at com.arthium.musica.audio.DesktopAudioPlayer$2.run(DesktopAudioPlayer.kt:27)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
`

**Comment by sedmelluq** [comment]
Fixed in `1.3.12`.
